projects/kit/directives/connected/connected.style.less

```diff
@@ -0,0 +1,40 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiConnected] {
+    --tui-icon-height: 2rem;
+    --tui-icon-width: 2rem;
+    --tui-left-offset: 1rem;
+    --tui-vertical-offset: calc((100% + var(--tui-icon-height)) / 2 + 4px);
+    --tui-connected-height: calc(100% - var(--tui-vertical-offset));
+
+    &[tuiCardLarge] [tuiCell],
+    [tuiCell],
+    [tuiStep] {
+        &:not(:first-of-type):before,
+        &:not(:last-of-type):after {
+            content: '';
+            position: absolute;
+            bottom: var(--tui-vertical-offset);
```

Comment:
I think we should move it out like top for after, otherwise it's not clear what wins for after element, top + bottom or height.

Reply:
should I take it out separately before and after or how? I didn't quite understand